packages/dicom-codec/README.md

```diff
@@ -26,138 +33,144 @@ yarn
 # Build
 yarn run build
 ```
+# How to use
+Commonjs
+```
+const dicomCodec = require('@cornerstonejs/dicom-codec');
+....
+const imageFrame = ....// see API
+const imageInfo = {} // add here image information (see API)
+const result = await dicomCodec.transcode(imageFrame, imageInfo, sourceTransferSyntaxUID, targetTransferSyntaxUID);
+const pixelData = dicomCodec.getPixelData(result.imageFrame, result.imageInfo);
+```
+ES6
+```
+import dicomCodec from '@cornerstonejs/dicom-codec';
+...
+const imageFrame = ....// see API
+const imageInfo = {} // add here image information (see API)
+const result = await dicomCodec.transcode(imageFrame, imageInfo, sourceTransferSyntaxUID, targetTransferSyntaxUID);
+const pixelData = dicomCodec.getPixelData(result.imageFrame, result.imageInfo);
+```
+
+
 
 # API
 
 ## decode
 
 async function that decodes an image
 
-Parameters:
-- compressedImageFrame - Uint8Array with the compressed image frame bytes
-- sourceTransferSyntaxUID - String with the transfer syntax uid of the compressed image frame
+Parameters (**It does not mutate any param**):
+- compressedImageFrame - TypedArray with the compressed image frame bytes.
 - imageInfo - Object
-    - rows - Number with the image rows/height
-    - columns - Number with the image columns/width
-    - bitsPerPixel - Number with bits per pixel
-    - componentsPerPixel - Number with number of components per pixel. 
-    - signed - Boolean true if pixel data is signed, false if unsigned
+    - rows - Number with the image rows/height.
+    - columns - Number with the image columns/width.
+    - bitsAllocated - Number with bits per pixel sample.
+    - samplesPerPixel - Number with number of components per pixel. 
+    - signed - Boolean true if pixel data is signed, false if unsigned.
+- sourceTransferSyntaxUID - String with the transfer syntax uid of the compressed image frame
 
 Returns:
 - Object
-    - imageFrame - Uint8Array with the uncompressed image frame bytes
+    - imageFrame - TypedArray with the uncompressed image frame bytes (Mostly codecs returns Uint8Array, but Uint16Array,Int16Array can be seen)
```

Comment:
What happens for single bit images?  That occurs for segmentations, so it would be nice to know.

Reply:
I think those normally aren't compressed